### PR TITLE
Add routes and change data response

### PR DESCRIPTION
**Pull request by brdunfield**
Added a couple routes, changed the routes to return JSON data. Updated all the ports to match the port being used in staging-tf

Added CORS for requests here as well. If we want to get rid of it later that's fine, it was just the quickest way to get to being able to test on my machine.

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from dotenv import load_dotenv
 
 from flask import Flask
+from flask_cors import CORS
```
Comment (sastels) [inline application.py:9]: getting a `ModuleNotFoundError: No module named 'flask_cors'` - did you install locally?

File: application.py [hunk 2]
```diff
@@ -66,10 +68,46 @@ def hello():
...
+        return(response)
     except:
-        return("Had an error connecting to database")
+        resp_error = {
+            'error':"Had an error connecting to database"
+        }
+        return(resp_error)
 
 
+@application.route('/live-services')
```
Comment (sastels) [inline application.py:102]: Just occurred to me, we should start a "Routes" section in the readme with a line for each of our routes.

**Review by sastels — CHANGES_REQUESTED** [review]

File: application.py (hunk 1 above)
Comment (brdunfield) [reply application.py:9]: oh, yes, I must have. I'll add it to requirements

File: requirements.txt [hunk 3]
```diff
@@ -63,21 +64,21 @@ alembic==1.4.2
...
 boto3==1.12.13
-botocore==1.17.30
+botocore==1.17.48
 certifi==2020.6.20
 chardet==3.0.4
 click==7.1.2
 colorama==0.4.3
 cryptography==3.0
-dnspython==1.16.0
+dnspython==2.0.0
```
Comment (sastels) [inline requirements.txt]: A bunch of other changes crept in during the `make freeze-requirements` including this one that was a problem before

File: application.py [hunk 4]
```diff
@@ -57,6 +57,24 @@ class Service(db.Model):
     def __repr__(self):
         return "<Service {} {}>".format(self.name, self.id)
 
+class Organisation(db.Model):
+    __tablename__ = "organisation"
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4, unique=False)
```
Comment (sastels) [inline application.py:63]: need an `import uuid` above for this line

**Review by sastels — CHANGES_REQUESTED** [review]
just need that uuid import, and maybe change back some of those imports.

**Comment by brdunfield** [comment]
Rolled back the dnspython, but aside from that it seems to work properly so we might as well keep the more up to date dependencies

**Review by sastels — APPROVED** [review]
🦑 🦑

**Pull request merged**